beps/0001-notifications-system/README.md

```diff
@@ -85,12 +85,43 @@ The role of the notifications plugin is to manage the lifecycle of notifications
 The notification backend stores notification using the [database service](https://backstage.io/docs/backend-system/core-services/index#database). In particular it needs to store the following information for each notification:
 
 - ID
-- Payload
+- Title
+- Priority
+- Origin
+- Topic (optional)
```

Comment:
Immediate thought on this is that it might make more sense to have this as custom metadata in the payload instead. This seems like something that would be fairly plugin/app specific and needs associated rendering logic in the frontend.

Reply:
right, this fits payload